plone/app/portlets/portlets/navigation.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from Acquisition import aq_inner, aq_base, aq_parent
 from ComputedAttribute import ComputedAttribute
+from plone import api
```

Comment:
Is that ok? @gforcada @jensens I do not remember what we decided about plone.api usage in core.